examples/examples-cpu/nyc-taxi-snowflake/xgboost-dask.ipynb

```diff
@@ -473,9 +471,11 @@
    "outputs": [],
    "source": [
     "test_dates = get_dates(\"2019-02-01\", \"2019-02-28\")\n",
-    "taxi_test = dd.from_delayed([load(conn_info, query, day) for day in test_dates])\n",
+    "taxi_test = dd.from_delayed([load(conn_info, day) for day in test_dates])\n",
     "taxi_test = taxi_test.persist()\n",
-    "_ = wait(taxi_test)"
+    "_ = wait(taxi_test)\n",
+    "\n",
+    "actual = taxi_test[y_col].compute()"
```

Comment:
Ok so I did this to get around the "Inputs contain futures that were created by another client" issue https://app.clickup.com/t/1250339/DEV-1699